README.md

```diff
@@ -30,7 +30,9 @@ that lives behind an OVS bridge and has its own unique interfaces.
 #### Step 1: Clone the project and bringup the VMs
 
 Note: if you have $GOPATH set, then please ensure either you unset GOPATH,
-or clone the tree in `$GOPATH/src/github.com/contiv/` location
+or clone the tree in `$GOPATH/src/github.com/contiv/` location. If you are
+using a Makefile target that needs `GOPATH`, please set it by doing 
```

Comment:
I don't think you need this change, it is covered by the first line.

Reply:
@rchirakk The first line is not clear IMO. It does not clearly say "you **MUST** set `GOPATH` for certain targets". Hence, @delapsley and I both saw issues without `GOPATH` as there is no check for empty `GOPATH` in `Makefile`. Since there is no check for empty `GOPATH`, I added this clear note for the user.